### PR TITLE
Change http://nodejs.org/ to https://nodejs.org/

### PR DESCRIPTION
**Pull request by bnoordhuis**
Bulk update HTTP links to their HTTPS counterpart.

### CODE REVIEW

File: locale/en/get-involved/index.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ right place. Explore our community resources to find out how you can help:
 
 ## International community sites and projects
 
-- [Australian Node.js blog &amp; developers listing](http://nodejs.org.au/)
+- [Australian Node.js blog &amp; developers listing](https://nodejs.org.au/)
```
Comment (phillipj) [inline locale/en/get-involved/index.md]: Was this intentional? That site doesnt seem to work with HTTPS

Comment (bnoordhuis) [inline locale/en/get-involved/index.md]: It wasn't.  I'll undo that before landing.

**Comment by rvagg** [comment]
sgtm

File: locale/en/blog/uncategorized/evolving-the-node-js-brand.md [hunk 2]
```diff
@@ -32,6 +32,6 @@ We look forward to exploring<ins cite="mailto:EMILY%20TANAKA-DELGADO" datetime="
 
 We hope you'll have fun using it.
 
-To download the new logo, visit <a href="http://nodejs.org/logos/">nodejs.org/logos</a>.
+To download the new logo, visit <a href="https://nodejs.org/logos/">nodejs.org/logos</a>.
```
Comment (fhemberger) [inline locale/en/blog/uncategorized/evolving-the-node-js-brand.md]: This redirects now to https://nodejs.org/static/images/logos/ and returns a 403 (directory index is forbidden).
https://nodejs.org/en/about/resources/ would be a better match, but direkt links to the logo files should still go to static/.

File: locale/en/blog/release/v0.10.0.md [hunk 3]
```diff
@@ -136,7 +136,7 @@ The benchmarks below were run on an Apple 13-inch, Late 2011 MacBook
 Pro with a 2.8 GHz Intel Core i7 processor, 8GB of 1333MHz DDR3 RAM,
 running OS X Lion 10.7.5 (11G63b).  The numbers are slightly different
 on Linux and SmartOS, but the conclusions are the same.  The [raw data
-is available](http://nodejs.org/benchmarks-v0.10-vs-v0.8/), as well.
+is available](https://nodejs.org/benchmarks-v0.10-vs-v0.8/), as well.
```
Comment (fhemberger) [inline locale/en/blog/release/v0.10.0.md:139]: 1. This must have come form a different source, as it wasn't part of the old nodejs.org website repo.

File: locale/en/blog/advisory-board/advisory-board-update.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ release of v0.12.
 The interim [advisory
 board](https://www.joyent.com/blog/node-js-advisory-board) has met three times
 since its creation. You can find the minutes from the advisory board meetings
-here: [http://nodejs.org/advisory-board](http://nodejs.org/advisory-board). As
+here: [https://nodejs.org/advisory-board](https://nodejs.org/advisory-board). As
```
Comment (fhemberger) [inline locale/en/blog/advisory-board/advisory-board-update.md]: Redirects to https://nodejs.org/en/about/organization/ (as there is no Advisory Board any more)

File: locale/en/blog/advisory-board/advisory-board-update.md [hunk 5]
```diff
@@ -59,7 +59,7 @@ project, and how does that team makes decisions.
 One thing that we all agree on, is that we're not going to be using the
 Benevolent Dictator model. In fact, recently the project hasn't been operating
 that way. We can be more clear about that in our
-[documentation](http://nodejs.org/about/organization). We all agree we want a
+[documentation](https://nodejs.org/about/organization). We all agree we want a
```
Comment (fhemberger) [inline locale/en/blog/advisory-board/advisory-board-update.md]: Redirects to https://nodejs.org/en/about/organization/

File: locale/en/blog/release/v0.10.0.md (hunk 3 above)
Comment (rvagg) [inline locale/en/blog/release/v0.10.0.md:139]: I could dig up the files for this but I think it'd be best to let it 404 at this stage, no harm in having it go via https though, it'd get redirected there anyway

File: locale/en/blog/community/building-nodejs-together.md [hunk 6]
```diff
@@ -28,17 +28,17 @@ living breathing website whose content is created by our end users and team.
 The website should be the canonical location for documentation on how to use
 Node.js, how Node.js works, and how to find out what's going on in the Node
 community. We have seeded the initial documentation with [how to
-contribute](http://nodejs.org/contribute), [who the core team
-is](http://nodejs.org/about/organization/#index_md_technical_steering_committee),
+contribute](https://nodejs.org/contribute), [who the core team
+is](https://nodejs.org/about/organization/#index_md_technical_steering_committee),
```
Comment (fhemberger) [inline locale/en/blog/community/building-nodejs-together.md]: Should probably link to https://nodejs.org/en/about/governance/

File: locale/en/blog/community/building-nodejs-together.md [hunk 7]
```diff
@@ -28,17 +28,17 @@ living breathing website whose content is created by our end users and team.
...
-itself](http://nodejs.org/about/organization). From there we're looking to
+itself](https://nodejs.org/about/organization). From there we're looking to
 enable the community to come in and build out the rest of the framework for
 documentation.
 
 One of the key changes here is that we're extending the tools that generate API
 documentation to work for the website in general. That means the website is now
 written in markdown. Contributions work with the same
-[pull-request](http://nodejs.org/contribute/code_contributions/) way as
+[pull-request](https://nodejs.org/contribute/code_contributions/) way as
```
Comment (fhemberger) [inline locale/en/blog/community/building-nodejs-together.md]: Should link to https://nodejs.org/en/get-involved/contribute/#code-contributions

File: locale/en/blog/community/building-nodejs-together.md [hunk 8]
```diff
@@ -28,17 +28,17 @@ living breathing website whose content is created by our end users and team.
 The website should be the canonical location for documentation on how to use
 Node.js, how Node.js works, and how to find out what's going on in the Node
 community. We have seeded the initial documentation with [how to
-contribute](http://nodejs.org/contribute), [who the core team
-is](http://nodejs.org/about/organization/#index_md_technical_steering_committee),
+contribute](https://nodejs.org/contribute), [who the core team
+is](https://nodejs.org/about/organization/#index_md_technical_steering_committee),
 and some basic documentation of the [project
-itself](http://nodejs.org/about/organization). From there we're looking to
+itself](https://nodejs.org/about/organization). From there we're looking to
```
Comment (fhemberger) [inline locale/en/blog/community/building-nodejs-together.md]: Redirects to https://nodejs.org/en/about/organization/

File: locale/en/blog/community/building-nodejs-together.md [hunk 9]
```diff
@@ -58,7 +58,7 @@ Road](http://blog.nodejs.org/2014/06/11/notes-from-the-road/) events there are
 often questions about what does and doesn't go into core. How the team
 identifies what those features are and when you decide to integrate them. I've
 spent a lot of time talking about that but I've also
-[added](http://nodejs.org/about/organization) it to the new documentation on
+[added](https://nodejs.org/about/organization) it to the new documentation on
```
Comment (fhemberger) [inline locale/en/blog/community/building-nodejs-together.md]: Redirects to https://nodejs.org/en/about/organization/

File: locale/en/blog/community/building-nodejs-together.md [hunk 10]
```diff
@@ -146,7 +146,7 @@ responsible for the entirety of that subsystem, but they guide its progress by
 communicating with end users, reviewing bugs and pull requests, and identifying
 test cases and consumers of new features. People come and go from the core
 team, and recently we've added [some
-documentation](http://nodejs.org/about/organization) that describes how you
+documentation](https://nodejs.org/about/organization) that describes how you
```
Comment (fhemberger) [inline locale/en/blog/community/building-nodejs-together.md]: Redirects to https://nodejs.org/en/about/organization/

File: locale/en/blog/community/building-nodejs-together.md [hunk 11]
```diff
@@ -28,17 +28,17 @@ living breathing website whose content is created by our end users and team.
 The website should be the canonical location for documentation on how to use
 Node.js, how Node.js works, and how to find out what's going on in the Node
 community. We have seeded the initial documentation with [how to
-contribute](http://nodejs.org/contribute), [who the core team
-is](http://nodejs.org/about/organization/#index_md_technical_steering_committee),
+contribute](https://nodejs.org/contribute), [who the core team
```
Comment (fhemberger) [inline locale/en/blog/community/building-nodejs-together.md]: https://nodejs.org/en/get-involved/contribute/

**Comment by fhemberger** [comment]
A few URLs could point directly to the new locations, otherwise LGTM.

**Comment by bnoordhuis** [comment]
Fixed up the links, PTAL.

**Comment by mikeal** [comment]
+1

**Comment by fhemberger** [comment]
Great, thank you!

**Pull request merged**